websites/R/Roblox Forum/presence.ts

```diff
@@ -25,8 +30,16 @@ presence.on("UpdateData", async () => {
 		presenceData.state = `Profile: ${document
 			.querySelector("meta[property='og:title']")
 			.getAttribute("content")}`;
-	} else if (document.location.pathname.startsWith("/whats-new/"))
-		presenceData.details = "Browsing latest content";
+		presenceData.buttons = [
+			{
+				label: "View Profile",
+				url: document.location.href,
+			},
+		];
+	} else if (window.location.pathname === ("/reports/" && "/approval-queue/"))
```

Comment:
This seems wrong or is it correct?

Reply:
do you read the things i said? i tried everything, it doesn't work. i might as well remove it